### ISSUE TITLE
Tutorial with large catalogs cross-matching

### ISSUE THREAD

**Issue by hombit** [astronomy-commons/lsdb#156]
We should have a basic tutorial about large catalog cross-matching covering all the steps needed to go from initial raw data to a directory with cross-matched hipscat catalog. So it should include something about `hipscat-import` and some recommendations about `Dask` configuration.

We may consider Gaia DR3 vs DES DR2 use-case. Gaia DR3 `gaia_source` CSV files are [here](http://cdn.gea.esac.esa.int/Gaia/gdr3/gaia_source/). DES DR2 FITS files are [here](https://desdr-server.ncsa.illinois.edu/despublic/dr2_tiles/), we can use only `*_dr2_main.fits` files for the main table data.

It also would be nice to have a "light" version of the notebook, which would run for a single file per catalog only, so users could run it on their laptops.

**Comment by hombit** [comment]
Blocked by https://github.com/astronomy-commons/hipscat-import/issues/227

**Comment by hombit** [comment]
Implemented in #166
